hibernate6-jpa/src/main/java/io/micronaut/configuration/hibernate6/jpa/JpaConfiguration.java

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.configuration.hibernate6.jpa;
```

Comment:
I would probably keep the old package name

Reply:
I wanted too, but some of the tasks was failing 
```
The :testCodeCoverageReport task failed.
Error while creating report
Error while analyzing /home/runner/work/micronaut-sql/micronaut-sql/hibernate6-jpa/build/classes/java/main/io/micronaut/configuration/hibernate/jpa/metrics/HibernateMetricsBinder.class with JaCoCo 0.8.8.202204050719/5dcf34a
Can't add different class with same name: io/micronaut/configuration/hibernate/jpa/metrics/HibernateMetricsBinder
```
because of the duplicated class names in existing hibernate-jpa and hibernate6-jpa

Reply:
I tried to disable that task but didn't find the way. Anyway, probably not good to have duplicate class names in different modules?

Reply:
@dstepanov Changed package name as we agreed so the report should pass now.

Reply:
Well, I hope it's final, changed package to be hibernate6 and avoid ambiguity. @dstepanov